examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
Per a discussion with @jordansissel I've significantly enriched the validation in `#register`. It now detects all sorts of crazy stuff AOT. Additionally, I added an 'auth' example, which is practically the only thing a user would probably set aside from headers.